ctparse/ctparse.py

```diff
@@ -105,6 +107,9 @@ def __init__(self, resolution, production, score):
         self.score = score
 
     def __repr__(self):
+        return f'CTParse({self.resolution}, {self.production}, {self.score})'
+
+    def __str__(self):
         return '{} s={:.3f} p={}'.format(self.resolution,
                                          self.score,
```

Comment:
I wanted to ask this question during the talk of Matti and James.
It seems to me that the new code changed relative ordering among resolution, production, and score.
But tests did not complain?
Thanks again for wonderful talk and conference and explaining the code.

Reply:
Which would probably point at a missing test szenario. But James `__repr__` is indeed more pythonic, as it reflects the `__init__`. Guess tests pass, since the `__repr__` is really only used for that: to represent